subsys/pm/policy.c

```diff
@@ -62,6 +62,10 @@ static struct k_spinlock events_lock;
 static sys_slist_t events_list;
 /** Next event, in absolute cycles (<0: none, [0, UINT32_MAX]: cycles) */
 static int64_t next_event_cyc = -1;
+/** Pointer to the next event. */
```

Comment:
Actually, it's not a pointer now. Is it intentional? Pointer would probably save a little memory